pkg/users/password-dialogs.js

```diff
@@ -148,7 +148,10 @@ export function password_quality(password) {
                     }
                 })
                 .fail(function(ex) {
-                    reject(new Error(ex.message || _("Password is not acceptable")));
+                    if (force)
+                        reject(new Error(ex.message || _("Password is not acceptable")));
```

Comment:
Why does this throw an exception when forcing is *on*? That looks the wrong way around.

Reply:
Yeah, the name was the wrong name around. Fixed